src/Uno.UWP/OnActivityResultArgs.Android.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Android.App;
+using Android.Content;
+
+namespace Uno
+{
+	/// <summary>
+	/// OnActivityResultArgs : an object containing the parameters of the Activity.OnActivityResult
+	/// </summary>
+	/// <param name="ct">CancellationToken</param>
+	/// <param name="intent">The Intent you want to send</param>
+	/// <param name="requestCode">A specific Response code, this is useful if you send more than one type of request in parallel</param>
+	public class OnActivityResultArgs
```

Comment:
```suggestion
	public class ActivityResultArgs
```

Also make this class internal